io/module_wrt_grid_comp.F90

```diff
@@ -21,7 +21,6 @@ module module_wrt_grid_comp
 !***  Revision history
 !***
 !     Jul 2017:  J. Wang/G. Theurich  - initial code for fv3 write grid component
-!     Aug 2017:  J. Wang              - add nemsio binary output for Gaussian grid
```

Comment:
This is really picky, but instead of removing tthe Aug 2017 note, you could add a Jul 2021 note that says "add nemsio binary output for Gaussian grid". Unless we think that this kind of "versioning" is no longer required in times of git (in which case we could remove all of it, of course)